modules/nf-core/deepvariant/tests/main.nf.test

```diff
@@ -0,0 +1,108 @@
+nextflow_process {
+
+    name "Test Process DEEPVARIANT"
+    script "../main.nf"
+    process "DEEPVARIANT"
+
+    tag "modules"
+    tag "modules_nfcore"
+
+    test("Test deepvariant") {
```

Comment:
Test names should be a little more descriptive from the [website](https://nf-co.re/docs/contributing/modules#migrating-from-pytest-to-nf-test):
``` 
Provide a test name preferably indicating the test-data and file-format used. Example: test("homo_sapiens - [bam, bai, bed] - fasta - fai")
```

Reply:
Modified as suggested

Reply:
it was an example from the docs, it wasn't a suggestion. I'm sorry for being unclear 😖

Reply:
Yep realised this and modified it again, no worries!